### PR TITLE
Add MTA space

### PR DESCRIPTION
**Pull request by dereksilva**

### CODE REVIEW

File: spaces/mstable/skin.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.mstable {
+  --primary-color: ##1065E0;
```
Comment (e-asia) [inline spaces/mstable/skin.scss:2]: You have 2 # in the hexadecimal color.
```suggestion
  --primary-color: #1065E0;
```

**Review by e-asia — CHANGES_REQUESTED** [review]

**Comment by dereksilva** [comment]
Changes made, thanks @e-asia

**Comment by bonustrack** [comment]
@e-asia We will wait final address later this week to merge this PR.

**Comment by e-asia** [comment]
@dereksilva @bonustrack I just noticed that the images are a bit too heavy. They should weigh 50bk max. Thank you for that.

**Comment by dereksilva** [comment]
Sorry, I did this yesterday but could only push the smaller images now.

**Review by e-asia — APPROVED** [review]

**Pull request merged**